src/server/game/Entities/Player/Player.cpp

```diff
@@ -12335,6 +12340,9 @@ void Player::MoveItemToInventory(ItemPosCountVec const& dest, Item* pItem, bool
     // only set if not merged to existing stack (pItem can be deleted already but we can compare pointers any way)
     if (pLastItem == pItem)
     {
+        pItem->SetFixedLevel(getLevel());
```

Comment:
Pretty sure this should never be called again after initial item creation (makes next line unneeded as well?)

Reply:
I put that in as a fallback measure, in case the player somehow received an already-created item that has not been locked to a level yet. Perhaps unclaimed loot from mail or something?

Reply:
unclaimed loot isnt implemented yet but it should already have it set (remember that item modifiers are sent in loot packet - you can see that on creature corpse already)